src/main/java/com/example/userservice/controller/UserController.java

```diff
@@ -0,0 +1,32 @@
+package com.example.userservice.controller;
+
+import com.example.userservice.dto.UserCreateRequest;
+import com.example.userservice.dto.UserCreateResponse;
+import com.example.userservice.service.UserService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/public/api/v1/users")
+@RequiredArgsConstructor
+public class UserController {
+
+    private final UserService userService;
+
+    @PostMapping
```

Comment:
по молодежному щас делать так 
```suggestion
    @PostMapping
    @ResponseStatus(HttpStatus.CREATED)
```